README.md

```diff
@@ -190,6 +190,21 @@ end
 
 Scripttags are created in the same way as the Webhooks, with a background job which will create the required scripttags.
 
+If you want to use the application.js file as generated by the Rails asset pipeline, you can do so by moving the `ShopifyApp.configure`
+to `config/application.rb` like so:
+
+```ruby
+config.after_initialize do
+	ShopifyApp.configure do |shopify_config|
+		shopify_config.scripttags = [
+			{event: 'onload', src: "https://example.com#{ActionController::Base.helpers.javascript_url('application')}"}
```

Comment:
I wonder if we can lazy init this inside the class as is rather than having to move the config. If this method isn't called until rails is initialized fully won't we get the fingerprint?